Add Meeting Notes from Scrum Meeting 04.04.2024 (#1245)
ElektraInitiative/PermaplanT: doc/meetings/2024_04_04_scrum_processes.md

# Meeting 04.04.2024 - Scrum Processes

## Participants

- Markus
- Andrei
- Yvonne

## Questions

The main questions to answer:

> - How can we improve the way we do issues?

Reviewer: ```suggestion
- How can we improve the way we create and work on issues?
```

> - How can we improve the way we deal with Reviews of PR

Reviewer: ```suggestion
- How can we improve the way we deal with reviews of PRs?
```

Takeaways:

- Every 3 weeks, include a 30min retro in the meeting by Andrei

> - After releases, have small Demo so team is up to date with what's new.

Reviewer: ```suggestion
- After releases, we have small demo so team is up to date with what's new.
```

- Explore how GitLab can help us structure issues around Use Cases/Features

> - Try to have only isses that have an estimation of one week (can be done in the scope of one "sprint") - If they are too big, they should be broken down into sub-tasks

Reviewer: ```suggestion
- Try to have only issues that have an estimation of one week (can be done in the scope of one "sprint") - If they are too big, they should be broken down into sub-tasks
```
